recipes/21cmFAST/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "21cmFAST" %}
+{% set version = "3.0.0.dev5" %}
```

Comment:
Please package fully released versions

Reply:
Ah. OK. That will have to wait for a little bit.

Reply:
Is there a 2.x you can package?

Reply:
No -- version 2 was a pure C library. v3 will represent the transition to a full Python-wrapped package.

Reply:
OK, I just updated it to point to v3.0.0

Reply:
... and all builds are passing, yay!